auth/password-visibility.js

```diff
@@ -0,0 +1,15 @@
+const visible = document.getElementsByClassName('visible');
+
+function passwordVisibility (e) {
+    if (e.target.getAttribute("src") === "/img/visibility_off.png") {
+        e.target.setAttribute("src", "/img/visibility_on.png");
+        e.target.parentElement.previousElementSibling.setAttribute("type", "text");
+    } else {
+        e.target.setAttribute("src", "/img/visibility_off.png");
+        e.target.parentElement.previousElementSibling.setAttribute("type", "password");
+    }
+}
+
+for (let icon of visible) {
```

Comment:
❗️ 수정요청
재할당하지 않을 경우 const로 선언해주세요~
```suggestion
for (const icon of visible) {
```